kasa/interfaces/light.py

```diff
@@ -25,11 +25,11 @@
 
 You can use the ``is_``-prefixed properties to check for supported features:
 
->>> light.is_dimmable
+>>> light.has_feature("brightness")
 True
->>> light.is_color
+>>> light.has_feature("hsv")
 True
->>> light.is_variable_color_temp
+>>> light.has_feature("color_temp")
```

Comment:
Should we do a breaking change and change to use more complete `color_temperature` wording while we are doing breaking changes?